modules/ROOT/pages/policy-mule3-simple-security-manager.adoc

```diff
@@ -0,0 +1,33 @@
+= Simple Security Manager Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Simple Security Manager policy lets you configure a security manager placeholder with hard-coded username and password for testing purposes.
+
+== Prerequisites
+
+This document assumes that you are an API Versions Owner for the API version that you want to manage, or that you are a member of the Organization Administrators role.
+
+== How Does This Policy Work?
+
+This policy acts as a simple security manager for you to test validations against your APIs using a hard-coded username and password.
```

Comment:
It's not exactly "hardcoded". It can be changed in the policy. It's more a 1-user-1-password strategy, that can be configured at policy level.